traffic_monitor/cache/stats_over_http.json_new

```diff
@@ -0,0 +1,539 @@
+{
```

Comment:
From Weasel:

```
Error                                 Unknown! traffic_monitor/cache/stats_over_http.json_new
```

Is `stats_over_http.json_new` needed? If it should stay, please add an exception for it in [`.dependency_license`](https://github.com/apache/trafficcontrol/blob/0321d20769/.dependency_license) to make Weasel happy.